docs/src/architecture/hardening.md

```diff
@@ -65,6 +68,28 @@ The host kernel runs on bare metal. The kernel is provided either with Linux ups
 
 #### `x86-64-linux`
 
-The host kernel hardening is based on Linux `make tinyconfig`. The default `tinyconfig` fails to assertions on NixOS without modifications. Assertions are fixed in `ghaf_host_hardened_baseline` Linux configuration under Ghaf `modules/host/`.
+The host kernel hardening is based on Linux `make tinyconfig`. The default `tinyconfig` fails to assertions on NixOS without modifications. Assertions are fixed in the `ghaf_host_hardened_baseline` Linux configuration under Ghaf `modules/host/`.
 
 In addition, NixOS (Ghaf baseline dependency) requires several kernel modules that are added to the config or ignored with `allowMissing = true`. As of now, the kernel builds and early boots on Lenovo X1.
+
+### Virtualization Support
+
+The host Virtualization support will add the required kernel config dependency to the Ghaf baseline by which NixOS has virtualization enabled. It can be enabled with the following flag `ghaf.host.kernel_virtualization_hardening.enable` for Lenovo X1.
+
+### Networking Support
+
+The host Networking support will add the required kernel config dependency to the Ghaf baseline by which NixOS has networking enabled, It can be enabled with the following flag `ghaf.host.kernel_networking_hardening.enable` for Lenovo X1.
+
+To enable GUI, set both Virtualization and Networking support. As of now, the kernel builds and can boot on Lenovo X1.
+
+### USB Support
+
+The host USB support will add the required kernel config dependency to the Ghaf baseline by which NixOS has USB enabled, It can be enabled with the following flag `ghaf.host.kernel_usb_hardening.enable` for Lenovo X1.
+
+USB support needs to be enabled in case of external SSD and is optional in case of interval NVME.
```

Comment:
Proposing:
USB support on host is for the `-debug`-profile builds, not for hardened host `-release`-builds. As of now, USB support needs to be enabled when debug support to host via USB ethernet adapter (Lenovo X1) is needed or when the user want to boot Ghaf using an external SSD. It is optional in case Ghaf is used with internal NVME.

Reply:
Updated documentation.